crates/sui-framework/tests/coin_balance_tests.move

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #[test_only]
-module sui::test_coin {
+module sui::test_coin_balance {
```

Comment:
just a rename this only tests coin balance, freeing it up to use `sui::test_coin` for more general coin tests.

Reply:
I think this should be `coin_balance_tests`? Not a strict requirement, but we try to keep 1 module per file, with the module name and file name matching.